Overview.src.html

```diff
@@ -2582,6 +2617,8 @@ <h5 id="calculating-the-scaled-active-time">Calculating the scaled active time</
         &times; <a lt="animation effect playback rate">playback
         rate</a> + <a>start offset</a></code>.
 
+        If <a>active time</a> is positive infinity and <a>active duration</a>
+
```

Comment:
This looks like a sentence fragment. Is there meant to be more, or is this meant to be another dt for the following dl?